src/e2e-tests/hero.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { test } from "@playwright/test";
+
+test("test2", async ({ page }) => {
+  await page.goto("https://hot.opensauced.pizza/discussed");
+  await page.locator("div:has-text(\"Find Open-Source Repositoriesto contribute today\")")
+    .nth(3)
+    .click();
+  await page.locator(
+    "body:has-text(\"OpenSaucedStar us on GitHubSign inFind Open-Source Repositoriesto contribute tod\")",
+  )
+    .press("Meta+k");
+  await page.getByPlaceholder("Search repositories")
+    .fill("test");
+  const [page4] = await Promise.all([
+    page.waitForEvent("popup"),
+    page.getByRole(
+      "link",
+      { name: "goldbergyoni/javascript-testing-best-practices goldbergyoni/javascript-testing-best-practices 📗🌐 🚢 Comprehensive and exhaustive JavaScript & Node.js testing best practices (December 2022) DouglasMV NoriSte Userbit js-kyle mel-mouk 49 19.9k" },
```

Comment:
can we have the testing repos be our owned repositories in https://github.com/orgs/open-sauced/repositories ?

Reply:
Yes, I can. 
Tnx.